gravitee-apim-gateway/gravitee-apim-gateway-standalone/gravitee-apim-gateway-standalone-distribution/pom.xml

```diff
@@ -104,6 +104,30 @@
             <scope>runtime</scope>
             <type>zip</type>
         </dependency>
+
+        <dependency>
```

Comment:
Those three dependencies are not required du to the fact there are present in `gravitee-apim-distribution` and so shared between mapi and gateway